### DISCUSSION TITLE
Multi exit supporter?

### DISCUSSION THREAD

**Discussion by abovzv** [PedestrianDynamics/jupedsim#1429] [Q&A]
![image](https://github.com/user-attachments/assets/8040216b-303b-4f02-aa4e-94ee9a3c7ad8)
I draw 2 exit sites, and make them a `Multipolygon`. But got error `RuntimeError: accessible area not connected` when add_exit_stage. Does it supported?

**Comment by abovzv** [comment]
And multi spawning_area...

**Comment by chraibi** [comment]
Yes, jupedsim supports multi exits. In fact, in this [notebook](https://github.com/PedestrianDynamics/CFM2/blob/main/notebooks/assignement2.ipynb) you can find a similar scenario are in your picture. 
It looks like this:
<img width="365" alt="Screenshot 2024-10-31 at 10 59 49" src="https://github.com/user-attachments/assets/4ea3da23-832a-46a6-8d39-aac8d0890f3e">

Maybe something is wrong with your geometry-config. You can post it here if you want. I can have a look.

**Reply by Ozaq** [reply]
The may only be a single polygon per exit stage. using a multiploygon is rejected (Albeit the error message looks like there is another issue)
